style.css

```diff
@@ -138,3 +163,539 @@ ul {
+  margin-left: 2px;
+  margin-top: 88px;
+}
+
+#work_3 {
+  display: grid;
+  height: 552px;
+  width: 327px;
+  margin-left: 2px;
+  margin-top: 88px;
+}
+
+#work_4 {
+  display: grid;
+  height: 596px;
+  width: 327px;
+  margin-left: 2px;
+  margin-top: 88px;
+}
+
+.project {
+  height: 220px;
+  width: 327px;
+  margin-top: 16px;
+  margin-left: 24px;
+  border-radius: 8px;
+}
+
+.project > img {
+  width: 80%;
+}
+
+.project_detail {
+  height: 36px;
+  width: 90%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-left: 24px;
+  margin-top: 12px;
+  color: #172b4d;
+}
+
+.job_detail {
+  width: 80%;
+  height: 32px;
+  top: 48px;
+  gap: 12px;
+  padding-left: 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 12px;
+  margin-left: 2px;
+}
+
+.canopy {
+  width: 55px;
+  height: 16px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  line-height: 16px;
+  text-decoration: none;
+  color: #344563;
+}
+
+.dot {
+  height: 8px;
+  width: 8px;
+  color: #7a869a;
+  line-height: 9px;
+}
+
+.job {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 11.9px;
+  line-height: 16px;
+  color: #7a869a;
+  width: 88px;
+  height: 16px;
+}
+
+.yrs {
+  width: 29px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.read {
+  width: 295px;
+  height: 76px;
+  top: 92px;
+  margin-left: 24px;
+  margin-top: 20px;
+}
+
+.read > p {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 18px;
+  color: #4b5975;
+}
+
+.progm_skills {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 203px;
+  height: 24px;
+  margin-top: 12px;
+  padding-left: 14px;
+}
+
+.skill_list1 > button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  border-color: #ebebff;
+  margin-left: -110%;
+  margin-top: 4px;
+}
+
+.skill_list2 > button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  border-color: #ebebff;
+  margin-left: -89%;
+  margin-top: 4px;
+}
+
+.skill_list3 > button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  border-color: #ebebff;
+  margin-left: -29%;
+  margin-top: 4px;
+}
+
+.seeProj {
+  width: 162px;
+  height: 48px;
+  margin-top: 12px;
+  border-radius: 8px;
+  padding-left: -12px;
+  text-align: center;
+}
```

Comment:
As the previous reviewer said, Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project. (_check the last project bulleted point in the image below_).You can check the desktop side of the figma design for the button effects. :pray:

![Screenshot from 2022-09-23 16-24-44](https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png)

Reply:
hover is not mandatory 
actually i put it by self but when reviewer told me use hover here so I removed that feature

Reply:
### I used ul li tag for html,css and js for this section
and earlier reviewer told me to use h tag in work_section bcz earlier I was using div.
i changed everything which she said 
except hover bcz that is not so compulsory.
few hours left for deadline,plz don't block for that reason.